JustSaying.Logging/LoggerExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Microsoft.Extensions.Logging;
+
+namespace JustSaying.Logging
+{
+    public static class LoggerExtensions

```

Comment:
Actually, I think I will inline this, cause I'm throwing out the rest of this project, this was just here to ease the migration. I'll do it as my last commit of this review to reduce the noise.